src/main/java/core/basesyntax/FigureGenerator.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureGenerator {
+    private static final String[] arrayOfFigures = new String[]{
```

Comment:
i don't usage of this variable in the code, do we need it somewhere?

Reply:
done